docs/tutorials/cooperative_groups_tutorial.rst

```diff
@@ -0,0 +1,312 @@
+.. meta::
+  :description: The tutorial of cooperative groups in HIP
+  :keywords: AMD, ROCm, HIP, cooperative groups, tutorial
+
+*******************************************************************************
+Tutorial: Cooperative Groups
```

Comment:
The current contents of this topic looks like this:
![image](https://github.com/ROCm/HIP/assets/164940351/4882a7d7-90e3-492e-884d-3ad95c33f82d)

Perhaps it should be
Prerequisites
Simple HIP Code
Tiled Partition
Device-side code
Host-side code
Conclusion

Reply:
I want to extend later with more tutorials. 

The Device-side code
Host-side code
Conclusion

are part of the Tiled partition.